CSS/style.css

```diff
@@ -0,0 +1,516 @@
+@font-face {
+    font-family: "Lato", sans-serif;
+        src: url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+}
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    font-family: Lato, Verdana, sans-serif;
+    font-size: 10px;
+    max-width: 1020px;
+    margin: auto;
+}
+
+h1 {
+    font-size: 2.5rem;
+}
+
+.header {
+    height: 89px;
+    background: #2D303A;
+    font-size: 2.5rem;
+    display: flex;
+    border-bottom-width: 6px;
+    border-bottom-color: #323746;
+    border-bottom-style: solid;
+    justify-content: space-between;
+}
+
+.logo {
+    padding: 29px 0 0 40px;
+    font-weight: 700;
+    line-height: 30px;
+    letter-spacing: -0.03em;
+    color: #FFFFFF;
+    width: 124px;
+    height: 25px;
+    text-transform: uppercase;
+}
+
+.star {
+    color: red;
+    font-size: 2.5rem;
+}
+
+.header-link-menu {
+    width: 375px;
+    height: 14px;
+    padding-right: 40px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.navigation-links {
+    display: inline-block;
+    margin-top: 42px;
+    font-size: 1.2rem;
+    font-weight: bold;
+    letter-spacing: -0.035em;
+    color: #FFFFFF;
+    position: relative;
```

Comment:
You can remove this position relative and implement .navigation-links::after with position relative instead of absolute.